cypress/e2e/auth_spec/redirect.cy.ts

```diff
@@ -0,0 +1,41 @@
+/// <reference types="cypress" />
+
+import { cy, describe, it } from "local-cypress";
+
+describe("redirect", () => {
+  it("Check if login redirects to the right url", () => {
+    cy.log("Logging in the user staffdev:Coronasafe@123");
+
+    cy.visit("/resource/board");
+    cy.get("input[id='username']").type("staffdev");
```

Comment:
loginPage.loginAsDisctrictAdmin(); is for district admin access, use the similar approach and create a staff login

Reply:
refer to the facility_homepage.cy.ts file to understand the POM approach we use in cypress

Reply:
The loginbyApi isn't ideal for current usecase since we need to test the redirect we need to login with manually inputting the credentials and having a loginAsStaffUser() which functions differently than the other two approaches is inconsistent i should probably create a LoginManuallyAsStaff() ?

Reply:
Yup, follow POM approach